pkg/resources/custom_metrics.go

```diff
@@ -18,22 +18,24 @@ import (
 )
 
 const (
-	sleepytime                              = 3600
-	DefaultPostgresMaintenanceMetricName    = "cro_postgres_service_maintenance"
-	DefaultPostgresInfoMetricName           = "cro_postgres_info"
-	DefaultPostgresAvailMetricName          = "cro_postgres_available"
-	DefaultPostgresConnectionMetricName     = "cro_postgres_connection"
-	DefaultPostgresStatusMetricName         = "cro_postgres_status_phase"
-	DefaultPostgresDeletionMetricName       = "cro_postgres_deletion_timestamp"
-	DefaultPostgresSnapshotStatusMetricName = "cro_postgres_snapshot_status_phase"
-	DefaultRedisMaintenanceMetricName       = "cro_redis_service_maintenance"
-	DefaultRedisInfoMetricName              = "cro_redis_info"
-	DefaultRedisAvailMetricName             = "cro_redis_available"
-	DefaultRedisConnectionMetricName        = "cro_redis_connection"
-	DefaultRedisStatusMetricName            = "cro_redis_status_phase"
-	DefaultRedisDeletionMetricName          = "cro_redis_deletion_timestamp"
-	DefaultRedisSnapshotStatusMetricName    = "cro_redis_snapshot_status_phase"
-	DefaultBlobStorageStatusMetricName      = "cro_blobstorage_status_phase"
+	sleepytime                                   = 3600
+	DefaultPostgresMaintenanceMetricName         = "cro_postgres_service_maintenance"
+	DefaultPostgresInfoMetricName                = "cro_postgres_info"
+	DefaultPostgresAvailMetricName               = "cro_postgres_available"
+	DefaultPostgresConnectionMetricName          = "cro_postgres_connection"
+	DefaultPostgresStatusMetricName              = "cro_postgres_status_phase"
+	DefaultPostgresDeletionMetricName            = "cro_postgres_deletion_timestamp"
+	DefaultPostgresSnapshotStatusMetricName      = "cro_postgres_snapshot_status_phase"
+	DefaultPostgresMaxAllocatedStorageMetricName = "cro_postgres_max_allocated_storage"
+	DefaultPostgresAllocatedStorageMetricName    = "cro_postgres_allocated_storage"
```

Comment:
This probably isn't going to scale well across clouds, but if it's something we need, let's go with it and create a ticket in the GCP Epic to be aware of these two metrics and how to satisfy them in GCP land

Reply:
good catch, will remove to have a single metric for `current_allocated_storage` that is what we will be alerting off of anyway :+1: